lib/rubocop/cop/rspec/spec_file_path_format.rb

```diff
@@ -73,7 +78,10 @@ def ensure_correct_file_path(send_node, class_name, arguments)
         def ignore_metadata?(arguments)
           arguments.any? do |argument|
             metadata_key_value(argument).any? do |key, value|
-              ignore_metadata.values_at(key.to_s).include?(value.to_s)
+              ignore_values = ignore_metadata[key.to_s]
+              ignore_values = [ignore_values] unless ignore_values.is_a?(Array)
```

Comment:
You can use `Array()` instead:
```suggestion
              ignore_values = Array(ignore_metadata[key.to_s])
```